### PR TITLE
Running `author` subcommand prints author information

### LINKED ISSUE #59: List authors of CLI when running `author|authors` sub command
This will make it easy for devs to find out who created a CLI

### PR DESCRIPTION
**Pull request by mfix22**
Hey @leo,
Let me know if you would like me to make updates. Here is an example output:

Output sample

Closes #59

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -612,6 +617,46 @@ class Args {
...
+
+    // get author(s) from parent module
+    const {authors, author, collaborators} = parent.exports
+
+    const sublist = (title, list) =>
+      Array.isArray(list) ?
+        [
+          chalk.underline(title),
+          ...list.map(a =>
+            `${chalk.dim('-')} ${chalk.yellow(a)}`)
```
Comment (ntwcklng) [inline index.js]: Can you use the new print Methods `this.printMailColor()` and `this.printSubColor()`?
https://github.com/leo/args/blob/master/index.js#L33

**Comment by mfix22** [comment]
@ntwcklng woops, forgot about that! Done ✔️.

**Comment by ntwcklng** [comment]
LGTM. Can you add a test?

**Comment by mfix22** [comment]
@ntwcklng cool! Let me know if you want it tested further. Just followed the `version` ones.
 
I could add something like this if you want: 
```
result = await run('author')
t.regex(result, new RegExp(author))
```

File: index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ class Args {
     this.config = {
       help: true,
       version: true,
+      author: true,
```
Comment (leo) [inline index.js]: Can we please call this `authors`?

**Comment by mfix22** [comment]
@leo cool! I updated it to work with `authors` too.  The menu looks like this: 
```
    ...
    author(s)  Display author info
    help       Display help
    ...
```
Let me know if you like the `(s)`, and if my solution is not too hack-y.

**Comment by leo** [comment]
Please use [command aliases](https://github.com/leo/args#commandname-description-init-aliases) 😊

**Comment by mfix22** [comment]
@leo shaking my head, missed that easy solution. My fault! Thanks for working with me.

File: readme.md [hunk 3]
```diff
@@ -58,6 +58,7 @@ In turn, this is how the auto-generated usage information will look like:
 
     serve, s       Serve your static site
     help           Display help
+    author         Display author info
```
Comment (leo) [inline readme.md]: We need to list the "authors" alias here.

File: readme.md [hunk 4]
```diff
@@ -176,6 +177,7 @@ By default, the module already registers some default options (e.g. "version" an
 | help | Automatically render the usage information when running `help`, `-h` or `--help` | true | Boolean |
 | name | The name of your program to display in help | Name of script file | String |
 | version | Outputs the version tag of your package.json | true | Boolean |
+| author  | Automatically render author information when running `author` | true | Boolean |
```
Comment (leo) [inline readme.md]: We need to list the "authors" alias here. In addition, the property is now called "authors", not "author".

**Comment by ntwcklng** [comment]
@mfix22 sorry for digging a huge hole under you work - could you add the `author` feature with the new file structure?

**Comment by mfix22** [comment]
@ntwcklng I am delighted to! Let me if you need anything else.

**Comment by ntwcklng** [comment]
Thank you!

**Comment by ntwcklng** [comment]
@leo could you approve the changes?

File: lib/authors.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+const pkginfo = require('pkginfo');
+
+module.exports = function() {
+  const parent = module.parent;
+
+  // Load parent module
+  pkginfo(parent);
+
+  // Get author(s) from parent module
+  const { authors, author } = parent.exports;
```
Comment (ntwcklng) [inline lib/authors.js]: @mfix22 can you make it work on node v4 without object destructuring?

**Comment by leo** [comment]
Oh. I completely forgot that there's `npm owners ls <package>`. So despite the effort you've put into this (thank you ❤️), we might not need it?

**Comment by ntwcklng** [comment]
@leo I wasn't even aware of `npm owner ls` 😅
Advantage of `<package> author`: it would blend in nicely and not just:
```
 $ npm owner ls args
leo <leo@zeit.co>
```

I would be happy to see this in prod.

**Comment by mfix22** [comment]
Either way works for me! Don't worry about it, just trying to knock off those issues and I believe either way does that!

**Comment by leo** [comment]
Thanks a lot for your effort. But I think we need to keep the codebase as small as possible. I sadly completely forgot about this npm command when I opened the issue... I would be very happy to accept a future PR of yours - but let's discuss it in the issue page first. ❤️

**Pull request closed without merging**